raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -36,7 +36,9 @@
     public static final String QUORUM_VOTERS_CONFIG = QUORUM_PREFIX + "voters";
     public static final String QUORUM_VOTERS_DOC = "Map of id/endpoint information for " +
         "the set of voters in a comma-separated list of `{id}@{host}:{port}` entries. " +
-        "For example: `1@localhost:9092,2@localhost:9093,3@localhost:9094`";
+        "For example: `1@localhost:9092,2@localhost:9093,3@localhost:9094.`" +
+        "If the voter endpoints are not known at startup, a non-routable address can be provided instead." +
```

Comment:
Perhaps we can keep this as an internal feature for now. It is not something that a user would be able to leverage. We can document it in the class javadoc perhaps.

Reply:
Fair enough. I can move it there.